lib/tags.js

```diff
@@ -73,6 +73,8 @@ exports.parseTagLine = (line) => {
  * @public
  *
  * @param {String[]} footerTagLines - footer tag lines
+ * @param {Object} [options={}] - options
+ * @param {Boolean} options.lowerCaseFooterTags - lowercase footer tag keys
```

Comment:
Do you think we should provide an optional value here? As it is now, this option is actually "optional", since if its not passed, its considered falsy. If that's OK, we can edit the JSDoc annotation to:

```js
@param {Boolean} [options.lowerCaseFooterTags=false] - lowercase footer tag keys
```

Otherwise, we should set:

```js
@param {Boolean} [options.lowerCaseFooterTags=true] - lowercase footer tag keys
```

and set a default below:

```js
_.defaults(options, {
    lowerCaseFooterTags: true
});
```